utils/categoryMapping.ts

```
@@ -0,0 +1,52 @@
+  "Construction": "Construction and the built environment",
+  "Engineering and Manufacturing": "Engineering and manufacturing",
+  "Creative and Design": "Creative and design",
+  "Public Safety and Emergency Services": "Protective services",
+  "Transportation and Logistics": "Transport and logistics",
+  "Agriculture, Environment, and Animal Care": "Agriculture, environmental and animal care"
+} as const;
+
+// Reverse mapping for converting database categories to display categories
+export const dbToCategoryMapping: { [key: string]: string } = 
+  Object.entries(categoryMapping).reduce((acc, [display, db]) => ({
+    ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)